libs/components/pages/src/lib/modules/modal-link-list/modal-link-list.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container
+  *ngIf="links === 'loading'; then loadingTemplateRef; else linksTemplateRef"
+></ng-container>
+
+<ng-template #headingTemplateRef>
+  <h2 class="sky-font-heading-4 sky-margin-stacked-sm">
+    {{ title }}
+  </h2>
+</ng-template>
+<ng-template #loadingTemplateRef>
+  <sky-wait [isWaiting]="true"></sky-wait>
+  <ng-container [ngTemplateOutlet]="headingTemplateRef"></ng-container>
+</ng-template>
+<ng-template #linksTemplateRef>
+  <ng-container *ngIf="links?.length > 0">
+    <ng-container [ngTemplateOutlet]="headingTemplateRef"></ng-container>
+    <ul
+      class="sky-modal-link-list"
+      [skyThemeClass]="{
+        'sky-margin-stacked-separate': 'default',
+        'sky-margin-stacked-xl': 'modern'
+      }"
+    >
+      <li
+        *ngFor="let link of links"
+        [skyThemeClass]="{
+          'sky-margin-stacked-default': 'default',
+          'sky-margin-stacked-sm': 'modern'
+        }"
+      >
+        <button
+          (click)="openModal(link)"
```

Comment:
The requirements in the story say "It should be able to open links and modals" - but this code seems to only allow for modals. Was this intentional?

Reply:
Good catch.